lib/ecto_ldap/adapter.ex

```diff
@@ -44,6 +48,29 @@ defmodule Ecto.Ldap.Adapter do
     {:reply, search_response, state}
   end
 
+  def handle_call({:update, schema_meta, fields, filters}, _from, state) do
```

Comment:
Not a map at all. The parameters that will be passed, unadulterated, to `:eldap/modify/3`.